### DISCUSSION TITLE
請問亞服可以使用嗎

### DISCUSSION THREAD

**Discussion by todomato** [lisai9093/YYS#5] [Q&A] [answered]
如題，謝謝

**Comment by lisai9093** [comment]
理论上来说大部分功能都是可以用的，但某些图片可能会不一样所以要稍作修改

**Comment by todomato** [comment]
抱歉，想再請教一下
我自己使用桌面版本都正常，
我想嘗試用ADB版本，抓完圖後，他在判斷位置的時候總是找不到
不知道您是否有遇過類似的問題呢?

謝謝

**Reply by lisai9093** [reply, marked as answer]
ADB我从来没有测试过，所以大概率是用不了的。。
